content-repo/extra-docs/articles/XSIAM_Playbooks.md

```diff
@@ -0,0 +1,148 @@
+---
+id: XSIAM-Playbooks
+title: XSIAM Alert handling Playbooks
+description: The XSIAM alerts handling playbooks included in this pack help you respond to Cortex XDR alerts in a timely manner. The playbooks are based on the MITRE ATT&CK tactics and techniques and the NIST framework Computer Security Incident Handling Guide.
+
+---
+The XSIAM alerts handling playbooks included in this pack help you respond to Cortex XDR alerts in a timely manner. The playbooks are based on the MITRE ATT&CK tactics and techniques and the NIST framework Computer Security Incident Handling Guide.
+
+## MITRE ATT&CK Tactics and Techniques
+MITRE ATT&CK is a globally-accessible knowledge base of adversary tactics and techniques based on real-world observations.
+The framework can be used by SOC and threat intelligence analysts, threat hunters, red teamers, and defenders to classify attacks better and assess risks for their organization.
+Organizations can use the framework to identify gaps in their defenses, prioritize them, and take the necessary actions to remediate the threat.
+Many cyber security systems classify incidents and reports based on the MITRE ATT&CK framework.
+Cortex XSOAR uses the MITRE ATT&CK feed integration to ingest the information about these techniques and sub-techniques and many different integrations to retrieve indicators and incidents obtaining these techniques.
+
+## NIST Framework - Computer Security Incident Handling Guide
+NIST's Computer Security Incident Handling Guide assists organizations in establishing computer security incident response capabilities and handling incidents efficiently and effectively. This publication provides guidelines for incident handling, particularly for analyzing incident-related data and determining the appropriate response to each incident.
+
+## XSIAM Alerts - BIOCs and ABIOCs
+Behavioral indicators of compromise (BIOCs) enable you to alert and respond to behaviors—tactics, techniques, and procedures. Instead of hashes and other traditional indicators of compromise, BIOC rules detect behavior such as is related to processes, registry, files, and network activity.
```

Comment:
> BIOC rules detect behavior such as is related to processes, registry, files, and network activity.

I think we should rephrase this sentence, for exmaple:
BIOC rules detect abnormal behaviors related to processes, registry keys, files, and network activity.